test/sql/local/irc/test_create_secret_attach_combinations.test

```diff
@@ -0,0 +1,180 @@
+# name: test/sql/local/irc/test_create_secret_attach_combinations.test
+# description: test combinations of create secret and attah
+# group: [irc]
+
+require avro
+
+require parquet
+
+require iceberg
+
+require httpfs
+
+# Test catalog secret and storage secret
+statement ok
+set enable_logging=true
+
+statement ok
+set logging_level='debug'
+
+require-env ICEBERG_SERVER_AVAILABLE
```

Comment:
nit: can we move this either to the top of the test file or above the first `statement ok`s?